index.html

```diff
@@ -2,236 +2,338 @@
+              <a href="#"><img src="./assets/icons/twitter.png" alt="" /></a>
+            </li>
+            <li>
+              <a href="#"><img src="./assets/icons/medium.png" alt="" /></a>
+            </li>
           </ul>
         </div>
-
-        <div class="framework">
-          <div class="dot">
-            <img class="rect" src="assets/square.png" alt="square" />
+        <div class="content">
+          <h1>Hey There.<br />I'm Chidiebere</h1>
+          <h2>I'am a Software Developer</h2>
+          <p>
+            I can help you build a product, feature or website Look through some
+            of my work and experience! If you like what you see and have a
+            project you need coded, <br />don't hesitate to contact me.
+          </p>
+        </div>
+        <div class="background"></div>
+      </section>
+      <section class="works">
+        <div class="title">
+          <h2>My Recent Works</h2>
+          <div class="separator"></div>
+        </div>
+        <div class="featured">
+          <div class="image">
+            <img
+              src="./assets/images/featured-placeholder.png"
+              alt="featured work"
+            />
+          </div>
+          <div class="content">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text
+              ever since the 1500s, when an unknown printer took a standard
+              dummy text.
+            </p>
+            <ul class="stacks">
+              <li><a href="#">css</a></li>
+              <li><a href="#">html</a></li>
+              <li><a href="#">bootstrap</a></li>
+              <li><a href="#">Ruby</a></li>
+            </ul>
+            <button class="btn btn-main">See Project</button>
           </div>
-          <ul class="frameElement">
-            <li class="lang">Framework</li>
-            <li class="boot">Bootstrap</li>
-            <li class="rubyRail">Ruby on Rails</li>
-            <li class="rspes">RSPes</li>
-            <li class="capybara">CapyBara</li>
-            <li class="sele">Selenium</li>
-          </ul>
         </div>
-
-        <div class="skill">
-          <div class="circle">
-            <img class="rect" src="assets/circle.png" alt="circle" />
+        <div class="cards-wrapper">
+          <div class="card">
+            <div class="content">
+              <h4 class="title">Profesional Art Printing Data More</h4>
+              <p class="summary">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required. has been the industry's standard
+              </p>
+              <ul class="stacks">
+                <li><a href="#">html</a></li>
+                <li><a href="#">bootstrap</a></li>
+                <li><a href="#">Ruby</a></li>
+              </ul>
+            </div>
```

Comment:
- [OPTIONAL] You did a very good job adding the hover of the card but I suggest sticking as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
Here you have a screenshot of your hover cards:

![image](https://user-images.githubusercontent.com/74506933/154802278-4999db86-5e9c-4162-8751-7ce24e94a227.png)

Here you have a screenshot of the design:

![image](https://user-images.githubusercontent.com/74506933/154802302-7ce21853-31b1-43b8-b3d8-e9444ab8769b.png)